README.md

```diff
@@ -397,9 +396,14 @@ tokenizeForagePinEditText.setForageConfig(
 - `ForageConfig.merchantID`: A unique seven digit numeric string that [FNS](https://docs.joinforage.app/docs/ebt-online-101#food-and-nutrition-service-fns) issues to authorized EBT merchants.
 - `ForageConfig.sessionToken`: A [session token](https://docs.joinforage.app/reference/create-session-token) that authenticates front-end requests to Forage. To create one, send a server-side request from your backend to the `/session_token/` endpoint.
 
-
 ### Step 4: Check the balance of the EBT Card
 
+> **FNS requirements for balance inquiries**
+>
+> FNS prohibits balance inquiries on sites and apps that offer guest checkout. Do not use this endpoint if your customers can opt for guest checkout.
```

Comment:
In this section, since we're referring to an SDK flow / Kotlin set of methods, so the term "endpoint" does not apply here.

```suggestion
> FNS prohibits balance inquiries on sites and apps that offer guest checkout. Skip this section if your customers can opt for guest checkout.
```